src/custom-components/EnhancedSelector.tsx

```diff
@@ -0,0 +1,268 @@
+import { SingleDropdownOption, Focusable } from '../deck-components';
```

Comment:
Why exactly reimplement the entire drop-down system? What benefit does this provide?